lib/run_argument_parser.py

```diff
@@ -177,19 +177,19 @@ def __init__(self):
             default=10
         )
         self.parser.add_argument(
-            '--normal-vaf', type=int,
+            '--normal-vaf', type=float,
             help="Normal VAF Cutoff. Sites BELOW this cutoff in normal will be considered. " +
             "Default: 0.02",
             default=0.2
```

Comment:
The default and the documented value don't match.

Reply:
(Seems like it ought to document the defaults for you...)

Reply:
Good catch. I fixed that. Unfortunately, the defaults don't appear when doing `--help`. Apparently this can be changed by setting the `formatter_class` argument of the `add_argument` to `argparse.ArgumentDefaultsHelpFormatter`.

This seems reasonable so I will make an issue to do that.